### ISSUE TITLE
*stream does not implement streammux.Stream

### ISSUE THREAD

**Issue by ericxtang** [whyrusleeping/go-smux-yamux#3]
Seems like this lib doesn't implement `streammux.Stream` anymore because of the new `Reset` method

**Comment by whyrusleeping** [comment]
Yeah, i'm working on the updates. Fell asleep last night before finishing all the deps, sorry about that!

**Comment by ericxtang** [comment]
No problem @whyrusleeping - just just happen to be setting up a new laptop, reverted to the old interface for now.

Appreciate all the good work you are putting out there :)

**Comment by Stebalien** [comment]
Fixed.
